src/AdminConsole/readme.md

```diff
@@ -39,17 +39,38 @@ The [Admin Console](https://admin.passwordless.dev/) is your primary GUI for cre
   },
   // https://docs.passwordless.dev/guide/self-hosting/configuration.html#e-mail
   "Mail": {
-    "Smtp": {
-      "From": "sender@example.com",
-      "Username": "username",
-      "Password": "password",
-      "Host": "smtp.example.com",
-      "Port": 587,
-      "StartTls": true,
-      "Ssl": false,
-      "SslOverride": false,
-      "TrustServer": false
-    }
```

Comment:
Don't we want to keep this as example of the SMTP email config?

Reply:
It is in the documentation, there are a lot of examples there